resources/META-INF/plugin.xml

```diff
@@ -31,7 +31,7 @@
   <change-notes></change-notes>
 
   <!-- please see http://www.jetbrains.org/intellij/sdk/docs/basics/getting_started/build_number_ranges.html for description -->
-  <idea-version since-build="145.0"/>
+  <idea-version since-build="193.0"/>
```

Comment:
There was a warning that FoldingDescriptor constructor is available only since 192.blahblah, so I just bumped since-build to 193 (who cares about older versions?)